components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/header.adoc

```diff
@@ -23,13 +23,14 @@ In Starknet, the block is defined as a list of transactions and a block header c
 | `extra data` | `FieldElement` | Extraneous data that might be useful for running transactions |
 |===
 
-[NOTE]
-====
-The commitment fields `event_commitment` and `transaction_commitment` are the roots of a height 64 binary Merkle Patricia tree.
+Where:
+
+
+[horizontal,labelwidth='20']
+`event_commitment`:: is the root of a height 64 binary Merkle Patricia tree. The leaf at index stem:[$i$] corresponds to the hash of the stem:[$i'th$] event.
+`transaction_commitment`:: is the root of a height 64 binary Merkle Patricia tree. The leaf at index stem:[$i$] corresponds to stem:[$h(transaction \ hash, signature)$] if the stem:[$i'th$] transaction is an `invoke` transaction and stem:[$h(0,0)$] otherwise.
```

Comment:
```suggestion
`transaction_commitment`:: is the root of a 64-bit high binary Merkle Patricia tree. The leaf at index stem:[$i$] corresponds to stem:[$h(transaction \ hash, signature)$] if the stem:[$i'th$] transaction is an `invoke` transaction and stem:[$h(0,0)$] otherwise.
```